redis/tests/test_async.rs

```diff
@@ -994,6 +994,56 @@ mod basic_async {
         .unwrap();
     }
 
+    #[test]
+    #[cfg(feature = "connection-manager")]
+    fn test_connection_manager_reconnect_after_delay_with_max_delay() {
```

Comment:
this is just a happy-path test, right? As far as I see the max delay isn't tested here.
If so, I don't the added benefit of this test, and IMO there's no need to add it. You can change the existing test to use `new_with_config` instead.

Reply:
random delay in a range so I can't test the exact number, I try to add some more cases

Reply:
if this isn't easily tested, the tests can be omitted.